MIGRATIONS.md

```diff
@@ -22,7 +74,8 @@ The following configuration options have been removed:
 +       site: 'datadoghq.eu'
     },
     telemetry: {
--       endPoint: 'https://app.datadoghq.eu'
+-       endPoint: 'https://app.datadoghq.eu',
+-       output: './metrics-debug',
```

Comment:
Is this change expected?

Reply:
Yes, the view can be misleading because it's a `diff` code block.

In the render:
<img width="1047" height="376" alt="Image" src="https://github.com/user-attachments/assets/cfddf178-a114-43d7-9ee0-365932171979" />

We do want to remove both `endPoint` and `output` options.